src/components/box/_box.scss

```diff
@@ -1,14 +1,22 @@
 $boxBorderColor: $graySecondary;
 $boxBorderRadius: 22px;
+$boxBorderSize: rhythm(1/8);
 
 @if ($include-html) {
 
   .mint-box {
     @include component;
-
+    display: inline-flex;
```

Comment:
`inline-flex` is used to:

* allow multiple `__hole`s in a single line
* prevents box from occupying the whole available horizontal space 
* allows holes to be positioned correctly.